scripts/sct_label_vertebrae.py

```diff
@@ -917,15 +917,15 @@ def label_discs(fname_seg_labeled, verbose=1):
     orientation_native = im_seg_labeled.change_orientation('RPI')
     nx, ny, nz = im_seg_labeled.dim[0], im_seg_labeled.dim[1], im_seg_labeled.dim[2]
     data_disc = np.zeros([nx, ny, nz])
-    vertebral_level_previous = np.max(im_seg_labeled.data)
-    # loop across z
+    vertebral_level_previous = np.max(im_seg_labeled.data)  # initialize with the max label value
+    # loop across z in the superior direction (i.e. starts with the bottom slice)
     for iz in range(nz):
         # get 2d slice
         slice = im_seg_labeled.data[:, :, iz]
         # check if at least one voxel is non-zero
         if np.any(slice):
             slice_one = np.copy(slice)
-            # set all non-zero values to 1
+            # set all non-zero values to 1 (to compute center of mass)
```

Comment:
Just curious, why not use center of mass with the original slice?

Reply:
if one slice includes part of vertebral level i and i+1, the center of mass will be shifted towards the i+1 level. We don't want that here (i.e. the goal is to be at the center of the cord)

Reply:
Thanks!